mopro-ios/MoproKit/Example/Tests/CircomUITests.swift

```diff
@@ -0,0 +1,13 @@
+import XCTest
+@testable import MoproKit_Example
+
+final class CircomUITests: XCTestCase {
+    
+    let ui = ViewController()
+
+    func testSuccessUI() {
+        XCTAssertNoThrow(ui.setupUI(), "Setup UI failed")
```

Comment:
We could have this, just keep in mind this will likely change and not sure UI tests are super useful for example app. Happy to leave it in though